### ISSUE TITLE
Can't generate any payloads: RPC EOF error

### ISSUE THREAD

**Issue by t1ndustries** [BishopFox/sliver#989]
Installed Sliver on a fresh Kali 2022.3 VM by using the linux one liner.

Upon generating any type of payload I receive this error:

![Screenshot_2022-11-03_16-49-12](https://user-images.githubusercontent.com/102918509/199841575-8c8ff4cf-b4e3-4bcc-a56c-41792280b379.png)

Checked ~/.sliver/logs/sliver.log and saw this:

![Screenshot_2022-11-03_16-56-41](https://user-images.githubusercontent.com/102918509/199842155-eefeb584-a772-42fd-b1af-cc5f7828f85d.png)

**Comment by moloch--** [comment]
Did this happen ounce, or every time your run the `generate` command?

**Comment by t1ndustries** [comment]
Last night, after shutting down my VM for the night, I accidentally reverted Kali back to its fresh install snapshot. This morning I went through the exact same process I did yesterday, apt updating and upgrading my Kali VM and installing Sliver via the one liner. Everything seems to be working fine now. I can generate payloads of any type without issue.
Unfortunately, I don't think I will be able to easily replicate this issue again.
